src/app/static/src/app/store/modelCalibrate/mutations.ts

```diff
@@ -29,10 +30,10 @@ export enum ModelCalibrateMutation {
     ComparisonPlotStarted = "ComparisonPlotStarted",
     SetPlotData = "SetPlotData",
     SetComparisonPlotData = "SetComparisonPlotData",
-    WarningsFetched = "WarningsFetched",
     CalibrateResultFetched = "CalibrateResultFetched",
     ClearWarnings = "ClearWarnings",
-    ResetIds = "ResetIds"
+    ResetIds = "ResetIds",
+    Metadata = "Metadata"
```

Comment:
This might be a little more consistent with the other mutation names:
```suggestion
    MetadataFetched = "MetadataFetched"
```